salt/modules/zypper.py

```diff
@@ -1029,7 +1087,8 @@ def mod_repo(repo, **kwargs):
         salt '*' pkg.mod_repo alias url= mirrorlist=http://host.com/
     '''
 
-    repos_cfg = _get_configured_repos()
+    root = kwargs.get('root', None)
+    repos_cfg = _get_configured_repos(root=root)
```

Comment:
This is a bug. 
```python
repos_cfg = _get_configured_repos(root=kwargs.get('root') or None)
```

Reply:
I need to assign to root, as is used later. Also I am not able to see the bug, as root will be None in case that is not there.

I changed the code to some proposal that was before, also I am not able to see the bug.